pg_search/src/postgres/build.rs

```diff
@@ -337,6 +339,10 @@ unsafe fn build_callback_internal(
                 .expect("could not register commit callbacks for build operation");
         });
         state.memctx.reset();
+
+        // important to count the number of items we've indexed for proper statistics updates,
+        // especially after CREATE INDEX has finished
+        state.count += 1;
```

Comment:
What are some examples of the statistics that would be influenced by this count? I'm wondering if I can add a test that checks them.

Reply:
```sql
CREATE TABLE foo (id SERIAL8, data TEXT);
INSERT INTO foo (data) SELECT md5(x) FROM generate_series(1, 1000000);
SELECT reltuples FROM pg_class WHERE oid = 'foo'::regclass;  -- reports -1
CREATE INDEX ...;  - if we set the state.count value, we can update it to 1000000
SELECT reltuples FROM pg_class WHERE oid = 'foo'::regclass; -- reports 1000000
```

It's to help make sure Postgres has a good initial estimate for the `pg_class.reltuples` column without first requiring an ANALYZE.

This value is good to have correct-ish as it's the value from which a selectivity percentage is usually calculated.

Reply:
I added a test for this.